packages/stateful/actions/core/dao_governance/AcceptSubDao/index.tsx

```diff
@@ -0,0 +1,76 @@
+import { FamilyEmoji } from '@dao-dao/stateless'
+import {
+  ActionComponent,
+  ActionKey,
+  ActionMaker,
+  UseDecodedCosmosMsg,
+  UseDefaults,
+} from '@dao-dao/types/actions'
+import { makeWasmMessage } from '@dao-dao/utils'
+
+import { AddressInput } from '../../../../components'
+import { AcceptSubDaoComponent, AcceptSubDaoData } from './Component'
+
+const defaultAcceptSubDaoData = {
+  address: '',
+}
+
+const useDefaults: UseDefaults<AcceptSubDaoData> = () => defaultAcceptSubDaoData
+
+const useDecodedCosmosMsg: UseDecodedCosmosMsg<AcceptSubDaoData> = (
+  msg: Record<string, any>
+) => {
+  try {
+    const match = Boolean(
+      msg.wasm.execute.contract_addr.msg.accept_admin_nomination
+    )
```

Comment:
```suggestion
    const match = Boolean(msg.wasm.execute.msg.accept_admin_nomination)
```

extraneous `contract_addr`